packages/jetpack-blocks/src/editor-shared/get-site-fragment.js

```diff
@@ -19,12 +20,14 @@ export default function getSiteFragment() {
 		return window._currentSiteId;
 	}
 
+	// FIXME: REMOVE THIS BLOCK! vvv
 	// Calypso will contain a site slug or ID in the site fragment.
 	// WP.org will contain either `post` or `post-new.php`.
 	const siteFragment = getCalypsoSiteFragment( window.location.pathname );
 	if ( ! includes( [ 'post.php', 'post-new.php' ], siteFragment ) ) {
 		return siteFragment || null;
 	}
+	// FIXME: REMOVE TO HERE ^^^
```

Comment:
Can we remove this stuff already now that Gutenlypso is out?

Reply:
Gutelypso is still available, albeit behind a feature flag (disable `calypsoify/iframe`). Until it's really removed and inaccessible, I think we need to continue full support.